### ISSUE TITLE
Figure out a way to merge curriculum content with RailsBridge SF

### ISSUE THREAD

**Issue by danchoi** [railsbridge-boston/railsbridge-boston#4]
cc: @alexch, @cykod, @mauricio 

We should try to figure out a good way to merge our fork of the curriculum with RailsBridge SF. It would be great to separate content from presentation and allow repurposing and reformatting the curriculum content in different ways to the fullest extent possible.

**Comment by mauricio** [comment]
That's true, we should probably see what sections overlap and how we could contribute it back to them.

**Comment by alexch** [comment]
Plus I love some of your improvements, like the "I've done this" checkbox
and slick table-of-contents styling.

**Comment by alexch** [comment]
cc @tjgrathwell @lilliealbert @nuclearsandwich
